support-frontend/assets/pages/supporter-plus-landing/components/threeTierCard.tsx

```diff
@@ -13,56 +13,48 @@ import {
 	LinkButton,
 } from '@guardian/source/react-components';
 import { BenefitsCheckList } from 'components/checkoutBenefits/benefitsCheckList';
-import type { Participations } from 'helpers/abTests/models';
 import type { RegularContributionType } from 'helpers/contributions';
 import { simpleFormatAmount } from 'helpers/forms/checkouts';
-import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
 import { currencies } from 'helpers/internationalisation/currency';
 import type {
 	Currency,
 	IsoCurrency,
 } from 'helpers/internationalisation/currency';
-import {
-	filterBenefitByABTest,
-	filterBenefitByRegion,
-	type ProductDescription,
-} from 'helpers/productCatalog';
 import type { Promotion } from 'helpers/productPrice/promotions';
 import { recurringContributionPeriodMap } from 'helpers/utilities/timePeriods';
+import type { LandingPageProductDescription } from '../../../helpers/globalsAndSwitches/landingPageSettings';
 import { ThreeTierLozenge } from './threeTierLozenge';
 
 export type ThreeTierCardProps = {
+	product: 'TierThree' | 'SupporterPlus' | 'Contribution';
 	cardTier: 1 | 2 | 3;
 	promoCount: number;
-	isRecommended: boolean;
-	isRecommendedSubdued: boolean;
+	isSubdued: boolean;
 	isUserSelected: boolean;
 	currencyId: IsoCurrency;
-	countryGroupId: CountryGroupId;
 	paymentFrequency: RegularContributionType;
 	link: string;
-	productDescription: ProductDescription;
+	productDescription: LandingPageProductDescription;
 	price: number;
 	ctaCopy: string;
 	lozengeText?: string;
 	promotion?: Promotion;
-	abParticipations?: Participations | undefined;
 };
 
 const container = (
-	isRecommended: boolean,
+	hasLabel: boolean,
```

Comment:
yep better than limiting to `isRecommended`

Reply:
Is this label the same label that may be renamed to pill in the benefitsChecklist? If so consider renaming here for consistency

Reply:
It's actually a separate pill that's displayed at the top of the card, but yes it's essentially the same thing.
And actually, in this file we also call it a "lozenge"!
I'm going to make a change to call it pill everywhere for consistency.
But in the model I'll leave it as `label` as I think it's nice for the model to not be too specific about the design